cli/vessel.js

```diff
@@ -0,0 +1,150 @@
+import {existsSync, mkdirSync, createWriteStream } from 'fs';
+import del from 'del';
+import { execaCommand} from 'execa';
+import chalk from 'chalk';
+import logUpdate from 'log-update';
+import { formatGithubDir, parseGithubURL, progressBar } from './mops.js';
+import path from 'path';
+import got from 'got';
+import decompress from 'decompress';
+import {pipeline} from 'stream/promises';
+
+const dhallFileToJson = async (filePath) => {
+	if (existsSync(filePath)) {
+		let cwd = new URL(path.dirname(import.meta.url)).pathname;
+		const res = await execaCommand(`dhall-to-json --file ${filePath}`, {preferLocal:true, cwd});
```

Comment:
Performance issues here. Each call takes 0.5 - 2 seconds

For a project with 2 deps (Itertools, serde + nested deps) it takes 10 seconds, which is unacceptably long. Each build will take +10 seconds.

Reply:
I see, There would be a huge delay when a lot more dependencies are added to a file.

The issue is probably because of the overhead caused by parsing the file using the `binary` file in the command line.

I think the solution would be to use a `dhall-to-json` lib that is written `js`.
I can't seem to find a `dhall-to-json` parser that is written in js. Do you think there is another way to reduce the parsing time?

I'm open to writing a `dhall-to-json` parser in js but that will take some time to implement.